### PR TITLE
Use latest dotnet channel version

### LINKED ISSUE #9: Use latest channel-version of dotnet
Use latest channel-version of dotnet instead of a fixed one.

[actions/setup-dotnet](https://github.com/actions/setup-dotnet) allows you to use the latest patch-version, so why not use it?

At the moment the [hardcoded version 3.1.201](https://github.com/github/ossar-action/search?q=dotnet-version%3A+%273.1.201%27&unscoped_q=dotnet-version%3A+%273.1.201%27) is used

### PR DESCRIPTION
**Pull request by litetex**
Fixes #9

### CODE REVIEW

**Comment by davidknise** [comment]
Use dotnet 3.1

File: .github/workflows/sample-workflow-ubuntu-latest.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       # Install dotnet, used by OSSAR
     - uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: '3.1.201'
+        dotnet-version: '3.1'
```
Comment (davidknise) [inline .github/workflows/sample-workflow-ubuntu-latest.yml:26]: According to [actions/setup-dotnet](https://github.com/actions/setup-dotnet), the syntax is `3.1.x` to use the latest channel version.

Comment (litetex) [reply .github/workflows/sample-workflow-ubuntu-latest.yml:26]: You can also use it without ``.x`` 😄 
All possibilities are ``3.1`` or ``3.1.x`` or ``3.1.*``

grafik

Comment (davidknise) [reply .github/workflows/sample-workflow-ubuntu-latest.yml:26]: That blurb does not make it clear that `3.1 == 3.1.x == 3.1.*`. I can run it to confirm, or the syntax listed on the action readme could be used.

image

You said this is "hardcoded," but it's not hardcoded, it's defaulted to what we build with and have tested. You are welcome to change it as you own your yaml that references the `setup-dotnet` action.

**Comment by litetex** [comment]
Seems to be fixed with https://github.com/github/ossar-action/pull/11

The files got changed and setup-dotnet is no longer used.
Also the Readme seems to be fixed.

Closing the PR

**Pull request closed without merging**